### DISCUSSION TITLE
Duplication of custom log/error message in policy validation output.

### DISCUSSION THREAD

**Discussion by hikkart** [open-policy-agent/community#37] [Gatekeeper] [answered]
Hi All,

Why do we get duplicate of our log message twice with **"Error from server" & "error when creating...."**

Is this expected behavior?? How to avoid duplicate?

Thanks in advance.

**Sample Output log:**

**Error from server** ([limit-podcount-per-node-policy] Limit exceeded for the given replica count: [7] ! Maximum allowed limit to scale is: [6]): **error when creating "frontend-deploy.yaml": admission webhook "validation.gatekeeper.sh" denied the request:** [limit-podcount-per-node-policy] Limit exceeded for the given replica count: [7] ! Maximum allowed limit to scale is: [6];

Our log **"([limit-podcount-per-node-policy] Limit exceeded for the given replica count: [7] ! Maximum allowed limit to scale is: [6]):"** comes twice in the output.

**Comment by srenatus** [comment, marked as answer]
https://github.com/open-policy-agent/gatekeeper/issues/278 looks like it was an issue in kubectl, and fixed in a more recent version.

**Reply by hikkart** [reply]
Thankyou @srenatus
